PumaGrasshopper/Utils.cs

```diff
@@ -322,5 +322,21 @@ public static string GetCastErrorMessage(string attribute, string castTarget)
         {
             return "Could not cast attribute " + attribute + " to " + castTarget;
         }
+
+        public static KeyValuePair<string, object> ParseInputPair(IGH_Goo input)
+        {
+            if (!input.CastTo(out string value)) throw new Exception("Could not cast input pair to string");
+
+            string[] tokens = value.Split(':');

```

Comment:
Also it seems like the values from an array are also split using a colon? https://github.com/Esri/puma/blob/13c5237419ce01ff07912996a3d32607d8919366/PumaGrasshopper/Utils.cs#L132

Does that work?

Reply:
I could use StringFromCeArray I think 👍